doc/azure2az_commands.rst

```diff
@@ -0,0 +1,305 @@
+azure storage blob copy start                                az storage blob copy start
+azure storage blob copy stop                                 az storage blob copy cancel
+azure storage blob delete                                    az storage blob delete
+azure storage blob download                                  az storage blob download
+azure storage blob lease acquire                             az storage blob lease acquire
+azure storage blob lease break                               az storage blob lease break
+azure storage blob lease change                              az storage blob lease change
+azure storage blob lease release                             az storage blob lease release
+azure storage blob lease renew                               az storage blob lease renew
+azure storage blob list                                      az storage blob list
+azure storage blob sas create                                az storage blob generate-sas
+azure storage blob show                                      az storage blob show
+azure storage blob snapshot                                  az storage blob snapshot
+azure storage blob update                                    az storage blob update
+azure storage blob upload                                    az storage blob upload
+azure storage container create                               az storage container create
+azure storage container delete                               az storage container delete
+azure storage container lease acquire                        az storage container lease acquire
+azure storage container lease break                          az storage container lease break
+azure storage container lease change                         az storage container lease change
+azure storage container lease release                        az storage container lease release
+azure storage container lease renew                          az storage container lease renew
+azure storage container list                                 az storage container list
+azure storage container policy create                        az storage container policy create
+azure storage container policy delete                        az storage container policy delete
+azure storage container policy list                          az storage container policy list
+azure storage container policy set                           az storage container policy update
+azure storage container policy show                          az storage container policy show
+azure storage container sas create                           az storage container generate-sas
+azure storage container set                                  az storage container set-permission
+azure storage container show                                 az storage container show
+azure storage cors delete                                    az storage cors clear
+azure storage cors set                                       az storage cors add
```

Comment:
As long as this leads them to the right function area, I think we are fine.  I didn't want to match exact functionality.   I'll call this example (and concept) in our migration document I'm writing